nixos/modules/services/networking/vpp.nix

```
@@ -0,0 +1,543 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib)
+    mkEnableOption
+    mkPackageOption
+    mkOption
+    types
+    mkIf
+    ;
+  cfg = config.services.vpp;
+
+  # Converts the settings option to a string
+  parseConfig =
+    cfg:
+    let
+      inherit (lib.lists) sort concatMap filter;
```

```suggestion
      inherit (lib) sort concatMap filter;
```
those are details we don't want to worry about